src/python/tests/storage/test_storage_api.py

```diff
@@ -18,6 +18,7 @@
 
 store = Storage("0.0.0.0", core_management_port=37251)
```

Comment:
This is clearly not always true. Please add a TODO here to fix this. My suggestion would be to force the storage service to use a fixed port of the tests. This can be done using the configuration for the storage service. But just add a TODO and we will resolve this in the next (or future) sprint.

Reply:
Yeah, we need to always pass this manually. 
Will add comment to FOGL-510 (valid for all tests fixture) itself as it will be for all the integration tests. I think better to force core to use fixed port in TEST env and then let the flow/ integration tested.